app/Jobs/CheckTraefikVersionJob.php

```
@@ -37,10 +37,29 @@ public function handle(): void
             return;
         }
 
+        $checkedAt = now()->toIso8601String();
+
         // Dispatch individual server check jobs in parallel
-        // Each job will send immediate notifications when outdated Traefik is detected
         foreach ($servers as $server) {
-            CheckTraefikVersionForServerJob::dispatch($server, $traefikVersions);
+            CheckTraefikVersionForServerJob::dispatch($server, $traefikVersions, false, $checkedAt);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Jobs\CheckTraefikVersionForServerJob' in method 'handle'. (undefined)

(StaticAccess)